code/modules/hydroponics/hydroponics.dm

```diff
@@ -254,19 +259,18 @@
 			pollinate()
 
 //This is where stability mutations exist now.
-			switch(myseed.instability)
-				if(100 to 80)
-					mutate(0, 0, 0, 0, 0, 0, 0, 2, 0) //Exceedingly low odds of gaining a trait.
-				if(79 to 60)
-					if(prob((myseed.instability)/2) && !self_sustaining)
-						mutatespecie()
-						myseed.instability = myseed.instability/2
-				if(40 to 59)
-					if(prob(40))
-						hardmutate()
-				if(20 to 39)
-					if(prob(40))
-						mutate()
+			if(myseed.instability >= 80)
+				mutate(0, 0, 0, 0, 0, 0, 0, 5, 0) //Exceedingly low odds of gaining a trait.
+			if(myseed.instability >= 60)
+				if(prob((myseed.instability)/2) && !self_sustaining) //Minimum 30%, Maximum 50% chance of mutating every age tick when not on autogrow.
+					mutatespecie()
+					myseed.instability = myseed.instability/2

```

Comment:
Suggestion: make this only cut the plant's instability in two if it actually mutated into a new kind of plant. Without that change, it'll basically become impossible to bring a plant up to 80+ instability for a chance to roll random traits without keeping the tray on self_sustaining mode at all times.

Reply:
Mutatespecie doesn't effect rolling random traits, it only effects species mutations. Also, due to the rolling mutation change, the only way to prevent species mutation for obtaining traits is autogrow as is. That said, I can make some changes inline with your prior review.

Reply:
@ATH1909 please avoid using the code review feature to make feature suggestions, and DO use the code review feature to suggest grammatical changes/code fixes, preferably with ctrl+g where applicable

Reply:
okay